style.css

```diff
@@ -0,0 +1,634 @@
+
```

Comment:
### Your Mobile Design
![image](https://user-images.githubusercontent.com/93039359/194261116-e859a64e-8d4a-40c0-bf29-8dd4f1a34936.png)

- [ ] Your mobile design is messed up :sob: You are not following the mobile first design approach. Mobile first design approach means that you create html and styling for mobile design. Then you create styling for desktop version using media queries in the same project. Kindly use media queries for your desktop design :+1:

Reply:
Fixed